__snapshots__/storybook.test.ts.snap

```diff
@@ -48387,2912 +48387,6 @@ exports[`Storyshots Trip Form Components Submode Selector 1`] = `
 </div>
 `;
 
-exports[`Storyshots TripDetails Bike Only Itinerary 1`] = `
```

Comment:
These should not be deleted.

Reply:
Snapshots reinstated and updated in https://github.com/opentripplanner/otp-ui/pull/281/commits/9725b4bf3d43303d383b1d42219fe199c3b7601e.